src/components/tables/DataTable.vue

```diff
@@ -116,7 +116,7 @@ export default {
       }
 
       if (dataCell.type && dataCell.type === 'dynamic') {
-        return $(node).find('p').text();
+        return node.dataset.value;
```

Comment:
I don't understand why `node` would refer to the `p` element that has the `data-value` attribute. I imagined `node` would refer to the `td` element.

Can you use `console.log(node)` to show me what `node` is referring to?